plugins/sentinelone/plugin.spec.yaml

```diff
@@ -2617,13 +2622,13 @@ actions:
         required: false
   update_analyst_verdict:
     title: Update Analyst Verdict
-    description: Updates an analyst verdict for each incident ID provided
+    description: Updates an analyst verdict for incident ID provided
     input:
-      incident_ids:
-        title: Incident IDs
-        description: A list of alert or threat IDs on which we may update the analyst verdict
-        type: "[]string"
-        example: ["1118189762920424575", "1118189762920424576"]
+      threat_id:
```

Comment:
I just wanted to check because there are two types of ID whether the name threat_id is applicable to both?

Reply:
Yes, the ID for incidents and for alerts can be the same